web/server/vue-cli/src/views/Reports.vue

```diff
@@ -140,7 +144,17 @@
           </v-chip>
         </template>
 
-        <template #item.reviewData="{ item }">
+
+        <template v-if="reportFilter.isUnique" #item.reviewData="{ item }">
+          <span v-if="sameReportsCounter != reports.length" />
```

Comment:
Why do you need this empty span item?

Reply:
This is definitely not a good idea. If I remove it, it works perferctly for me. If it not works for you, please comment the async call which set the dictionary and create a screenshot about the bad behavior.

A problem can be that `sameReports[item.bugHash]` is undefined before the API respond arrive but as far as I see vue handles this when used in v-for loop. If still not working we can do the following:
```
<template v-if="sameReports[item.bugHash]">
  <review-status-icon ...>
</template>
```

Also remove the `sameReportsCounter`.